DesignSafe-CI/portal
# Bug/DES-1795 - Fix duplication of authors on citation modal

## Overview: ##
There was a bug which was showing a duplicate list of authors on the citation modal for V2 projects
(bug does not affect other)

## PR Status: ##


## Related Jira tickets: ##

* [DES-1795](https://jira.tacc.utexas.edu/browse/DES-1795)

## Summary of Changes: ##

## Testing Steps: ##
1. Navigate into various published projects and check that the citation modal does not have duplicated lists of authors anymore (see photos)

## UI Photos:
1. Image of Bug in modal
![bug_citation](https://user-images.githubusercontent.com/29575979/97472964-69362e00-1918-11eb-8ab8-9d83829b9592.png)
2. Image of changes in PR
![fix_citation](https://user-images.githubusercontent.com/29575979/97472982-6dfae200-1918-11eb-9cb2-67f0e27c91a4.png)


## Notes: ##
